manage.py

```diff
@@ -1,12 +1,20 @@
 import os
+import bugsnag
 
 from flask_script import Manager, Shell
+from bugsnag.flask import handle_exceptions
```

Comment:
Since these are essentially bugsnag configurations, would you consider putting them in the Config.py file under ProductioConfig class